go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/influxdata/influxdb v1.6.1 // indirect
 	github.com/julienschmidt/httprouter v1.1.0
 	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
+	github.com/kr/pretty v0.2.0 // indirect
```

Comment:
I don't think you're using the changes in go.mod and go.sum.

Reply:
I believe the `//indirect` comment indicates that dependency is used by one of the other modules that we depend on.
```
The indirect comment indicates a dependency is not used directly by this module, only indirectly by other module dependencies. See go help modules for details.
```

Ref: https://blog.golang.org/using-go-modules

Reply:
The vim-go plug-in added them automatically, so I assumed they were needed.

Reply:
Ah, ok. I missed the indirect comment.